octoprint_octolapse/render.py

```diff
@@ -15,6 +15,7 @@
 import time
 
 import octoprint_octolapse.utility as utility
+from octoprint_octolapse.snapshot import CaptureSnapshot
```

Comment:
Instead of creating a new instance of CaptureSnapshot, why don't we take care of cleaning in timelapse.py within the event handlers (onRenderComplete/fail/etc)?  That way we can add a bunch more cleaning options (start of print, end of print, etc) if we feel like it.  Also, I think there is already an instantiation of that class in timelapse.py.

Reply:
I'll check it out